tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "./tsconfig.base.json",
+  "files": [],
+  "references": [
```

Comment:
This is the meat of the global typescript setup for the mono repo. It allows one tsc command to transpile all sub-packages.